types/ember/index.d.ts

```diff
@@ -328,7 +167,128 @@ interface RenderOptions {
+    */
+    classNameBindings: string[];
+    classNames: string[];
+}
+const ClassNamesSupport: ClassNamesSupport;
+
+/**
+TextSupport is a shared mixin used by both Ember.TextField and Ember.TextArea. TextSupport
+adds a number of methods that allow you to specify a controller action to invoke when a
+certain event is fired on your text field or textarea. The specifed controller action would
+get the current value of the field passed in as the only argument unless the value of the field
+is empty. In that case, the instance of the field itself is passed in as the only argument.
+**/
+interface TextSupport extends TargetActionSupport {
+    cancel(event: Function): void;
+    focusIn(event: Function): void;
+    focusOut(event: Function): void;
+    insertNewLine(event: Function): void;
+    keyPress(event: Function): void;
+    action: string;
+    bubbles: boolean;
+    onEvent: string;
+}
+const TextSupport: Ember.Mixin<TextSupport, Ember.Component>;
+
+/**
+Ember.ActionHandler is available on some familiar classes including Ember.Route,
+Ember.Component, and Ember.Controller. (Internally the mixin is used by Ember.CoreView,
+Ember.ControllerMixin, and Ember.Route and available to the above classes through inheritance.)
+**/
+interface ActionHandler {
+    /**
+    Triggers a named action on the ActionHandler. Any parameters supplied after the actionName
+    string will be passed as arguments to the action target function.
+
+    If the ActionHandler has its target property set, actions may bubble to the target.
+    Bubbling happens when an actionName can not be found in the ActionHandler's actions
+    hash or if the action target function returns true.
+    **/
+    send(actionName: string, ...args: any[]): void;
+    /**
+    The collection of functions, keyed by name, available on this ActionHandler as action targets.
+    **/
+    actions: ActionsHash;
+}
+const ActionHandler: Ember.Mixin<ActionHandler>;
+
+interface TriggerActionOptions {
+    "action"?: string;
+    "target"?: Ember.Object;
+    "actionContext"?: Ember.Object;
+}
+/**
+Ember.TargetActionSupport is a mixin that can be included in a class to add a triggerAction method
+with semantics similar to the Handlebars {{action}} helper. In normal Ember usage, the {{action}}
+helper is usually the best choice. This mixin is most often useful when you are doing more
+complex event handling in Components.
+**/
+interface TargetActionSupport {
+    triggerAction(opts: TriggerActionOptions): boolean;
+}
+
+/**
+Additional methods for the Controller.
+**/
+interface ControllerMixin extends ActionHandler {
+    replaceRoute(name: string, ...args: any[]): void;
+    transitionToRoute(name: string, ...args: any[]): void;
+    model: any;
+    queryParams: string[] | Array<{ [key: string]: { type: string } }>;
+    target: Object;
+}
+const ControllerMixin: Ember.Mixin<ControllerMixin>;
+
+namespace Ember {
```

Comment:
@theroncross The tests pass locally for me if you change this line back to `export namespace Ember` and revert the ember-inflector tests changes in 5356bd02f85cbf0b827e3e8006819ab7ca7f4adb

Reply:
Got it. I'm still getting `Cannot find name..`s in the `ember-qunit` declarations for Hooks, Assert, and Qunit. I've diffed everything I can think of to see if I've accidentally changed something. Where are those declared?

Reply:
What code are you referring to?